config/crd/bases/kubeflow.org_tfjobs.yaml

```diff
@@ -0,0 +1,7355 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.1
+  creationTimestamp: null
+  name: tfjobs.kubeflow.org
```

Comment:
No, it's auto-generated when I execute `make generate` & `make manifests` after adding XGBoost API. Because these two command in Makefile apply code generation for all files in `pkg/apis/...`, the TFJob, whose api files are already there since we didn't remove it, will also be added to the code-generation phase.

Reply:
Got you. We don't necessary need to check it in. This is minor and thanks for explaination